etc/sudoers.d/spinnaker

```diff
@@ -0,0 +1,2 @@
+# User rules for spinnaker
+spinnaker ALL=(ALL) NOPASSWD:ALL
```

Comment:
sgtm

On Sun, Mar 20, 2016 at 11:35 AM Eric Wiseblatt <notifications@github.com>
wrote:

> In etc/sudoers.d/spinnaker
> <https://github.com/spinnaker/spinnaker/pull/816#discussion_r56766327>:
>
> > @@ -0,0 +1,2 @@
> > +# User rules for spinnaker
> > +spinnaker ALL=(ALL) NOPASSWD:ALL
>
> right. I dont know if he needs anything else. Even giving packer root
> access is scary because packer has powerful scripting in it, but
> restricting to packer keeps the other services from gaining root so this is
> definitely a significant improvement. I might still get other
> recommendations on Monday so lets give it another day.
> … <#m_-7705907150762289740_>
> On Sun, Mar 20, 2016 at 10:44 AM, Matt Duftler <notifications@github.com>
> wrote: In etc/sudoers.d/spinnaker <#816 (comment)
> <https://github.com/spinnaker/spinnaker/pull/816#discussion_r56765496>>:
> > @@ -0,0 +1,2 @@ > +# User rules for spinnaker > +spinnaker ALL=(ALL)
> NOPASSWD:ALL If we change this to the following, it will only allow sudo
> for the packer command: spinnaker ALL=(ALL) NOPASSWD: /usr/bin/packer — You
> are receiving this because you were mentioned. Reply to this email directly
> or view it on GitHub <
> https://github.com/spinnaker/spinnaker/pull/816/files/2130ca5466470e30472ce6fe935171247e6f034f#r56765496>
>
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/spinnaker/spinnaker/pull/816/files/2130ca5466470e30472ce6fe935171247e6f034f#r56766327>
>